tools/update-npm.sh

```diff
@@ -42,6 +42,11 @@ echo "Copying new npm"
 
 tar zxf "$WORKSPACE"/cli/release/npm-"$NPM_VERSION".tgz
 
+echo "Building semver"
+
+cd npm/node_modules/semver
+"$BASE_DIR"/out/Debug/node ../../bin/npx-cli.js esbuild@0.14.38 index.js --bundle --platform=node --outfile=semver.js
```

Comment:
In other files, we are using the following pattern when trying to call npm: https://github.com/nodejs/node/blob/df5e82bfd1ae74840677d9a1fd64e5edbbd2bfa3/tools/update-eslint.sh#L17-L22
We should probably use it here for consistency.